vagrant-kubernetes-istio/RunTestOnHost/README.md

```diff
@@ -67,6 +67,23 @@ cd $ISTIO/istio
 make e2e_simple E2E_ARGS="--use_local_cluster"
 ```
 
+### 3. Debug with Delve
+Assume you have completed [Run tests on VM](https://github.com/istio/istio/tree/vagrant-kubernetes-istio/vagrant-kubernetes-istio/RunTestOnHost#2-run-tests-on-vm), now you can try debug Istio with debugger tool. vm_setup.sh already installs Delve into VM for us. To use Delve, we need process id of the binary we want to debug.
```

Comment:
This is true for both cases...runtestonhost and runtestonvm

Reply:
I have updated the file.